nixos/modules/services/networking/hostapd.nix

```diff
@@ -1,222 +1,1283 @@
 { config, lib, pkgs, utils, ... }:
+# All hope abandon ye who enter here. hostapd's configuration
+# format is ... special, and you won't be able to infer any
+# of their assumptions from just reading the "documentation"
+# (i.e. the example config). Assume footguns at all points -
+# to make informed decisions you will probably need to look
+# at hostapd's code. You have been warned, proceed with care.
+let
+  inherit
```

Comment:
Why do we maintain this list of lib functions?

Reply:
Yeah, I'd say to just use `lib.` everywhere, it's just three letters anyway.

Reply:
Is this a relevant change though?

Since some consider [`with lib` to be an antipattern](https://nix.dev/recipes/best-practices#with-scopes) I chose this to make the code more readable without having `lib.` bloating the line everywhere. Since serveral files in nixpkgs do the same thing I didn't expect this to be controversial.

Reply:
Let's leave those nitpicks out of the PR as it's already quite hairy, complicated and probably taking a lot of patience from the author. :)